hat/conf/v26.routes

```diff
@@ -70,7 +70,7 @@ GET           /she/function
 GET           /she/function/:function                                                        org.hatdex.hat.she.controllers.FunctionManager.functionGet(function: String)
 GET           /she/function/:function/enable                                                 org.hatdex.hat.she.controllers.FunctionManager.functionEnable(function: String)
 GET           /she/function/:function/disable                                                org.hatdex.hat.she.controllers.FunctionManager.functionDisable(function: String)
-GET           /she/function/:function/trigger                                                org.hatdex.hat.she.controllers.FunctionManager.functionTrigger(function: String)
+GET           /she/function/:function/trigger                                                org.hatdex.hat.she.controllers.FunctionManager.functionTrigger(function: String, useAll: Option[Boolean])
```

Comment:
There is a way to have 1 url for both cases. Just flagging

Reply:
I am not sure what you mean?

Reply:
`useAll: Option[Boolean]` can become `useAll: Boolean ?= false` and you avoid the option part. I am saying that only because, if I am correct, you are defaulting to false.  
https://www.playframework.com/documentation/2.8.x/ScalaRouting#Parameters-with-default-values

Reply:
Good point, updated in the new commit